book-crawler/app/crawler/kyobo.py

```
@@ -155,43 +195,20 @@ def crawl_book_detail(driver: webdriver.Chrome, url: str) -> dict | None:
-                        
-                        return false;
-                    """)
-                    
-                    if moved:
-                        time.sleep(2)
-                    else:
+                    # 페이지네이션 버튼 찾기
+                    next_button = driver.find_element(By.XPATH, f"//div[@class='pagination']//a[text()='{review_page + 1}']")
+                    driver.execute_script("arguments[0].click();", next_button)
+                    human_delay(base=2.0, spread=1.0)
+                except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)